.env.example

```diff
@@ -1,3 +1,9 @@
 NODE_ENV=
 PORT=
-DATABASE_URL=
\ No newline at end of file
+DATABASE_URL=
+LOCAL_DB_USER=
+LOCAL_DB_PASSWORD=
+LOCAL_DB_NAME=
+TEST_DB_USER=
+TEST_DB_PASSWORD=
+TEST_DB_NAME=
```

Comment:
Could consider using a  database `connection URI` instead of saving a lot of DB credentials in `.env`

Reply:
Okay I am on it